src/iop/colorbalance.c

```diff
@@ -1120,13 +1139,6 @@ static void apply_gain_auto(dt_iop_module_t *self)
 }
 #endif
 
-static void disable_colorpick(struct dt_iop_module_t *self)
```

Comment:
@aurelienpierre : I suppose all the changes below are not expected. right?

Reply:
Yes, "git reset -p HEAD^" but this is not on the top commit so you also need to play with git rebase -i I suppose. I can handle this in less than a minute, so I can do it if you don't feel messing with Git. Just let me know.

Reply:
Ok I let you do it. Thank you !

Reply:
Done, please double check that I haven't mess anything. Thanks.

Reply:
Looks good. I will test later.